src/components/FileDetail/FileDetail.js

```diff
@@ -52,10 +49,18 @@ const FileDetail = ({fileNode, history, match}) => {
                     <p className="FileInfo--Title">File Type:</p>
                     <div className="flex">
                       <div className="FileInfo--Icon">
-                        {fileNode.fileType === 'SHM' && <Box />}
-                        {fileNode.fileType === 'CLN' && <Clinical />}
-                        {fileNode.fileType === 'SEQ' && <Sequencing />}
-                        {fileNode.fileType === 'OTH' && <Misc />}
+                        {fileNode.fileType === 'SHM' && (
```

Comment:
can use the same map as above ☝️ maybe just move the `iconKind` into the `/common` folder as a global util